docs/user_guide.md

```diff
@@ -24,10 +26,10 @@ $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10
 ```
 
 Argument List:
-- config: Mandatory. The path to the CSV config file.
-- output: Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
-- since : Optional. start date of analysis. Format: dd/MM/yyyy
-- until : Optional. end date of analysis. Format: dd/MM/yyyy
+- config : Mandatory. The path to the CSV config file.
+- output : Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
+- since : Optional. start date of analysis. Format: `DD/MM/YYYY`
+- until : Optional. end date of analysis. Format: `DD/MM/YYYY`
```

Comment:
why you use big `DD` and `MM` :bomb: `D` is the full date, `M` is minute.

by convention, small `d` is date, and `m` is month, `Y` is alright.
[strf ref](http://www.cplusplus.com/reference/ctime/strftime/)

Reply:
User-guide should be free from programming syntax. So common convention should be fine.